### DISCUSSION TITLE
簡単な起動方法

### DISCUSSION THREAD

**Discussion by chiyu-maeda** [raichiii/russell-evaluation#12] [Q&A] [answered]
dockerの起動あたりがよく分かりません。
あまりPCを扱ったことがない人でも簡単に実行できるいい方法はありますか？

**Comment by raichiii** [comment, marked as answer]
start.exampleファイル（Shellスクリプト）を作成したので、startにコピー後、デスクトップにシンボリックリンクを置いてください
